llvm/unittests/Support/PrefixMapperTest.cpp

```diff
@@ -114,8 +136,8 @@ TEST(PrefixMapperTest, construct) {
 
 TEST(PrefixMapperTest, add) {
   PrefixMapper PM(sys::path::Style::posix);
-  PM.add(MappedPrefix{"a", "b"});
-  PM.add(MappedPrefix{"b", "a"});
+  cantFail(PM.add(MappedPrefix{"a", "b"}));
```

Comment:
Should we use `ASSERT_THAT_ERROR(..., Succeeded())` for tests?  `cantFail` is only an `assert` so it could fail silently in a no-asserts build.

Reply:
Done 👍